config/canonical.json

```diff
@@ -14846,6 +14846,18 @@
       "name": "Frankie & Benny's"
     }
   },
+  "amenity/restaurant|Freshii": {
+    "match": ["amenity/fast_food|Freshii"],
+    "nocount": true,
+    "tags": {
+      "amenity": "fast_food",
```

Comment:
So then this line should be `"amenity": "restaurant",` to match with line 14849 correct? Since your matching with fast_food but the actual tag being used is amenity=restaurant.

Reply:
Oh yep that is a mistake. I didn't cut that over to restaurant. 🤦‍♂️

Reply:
Ah well, that explains it then. I knew it looked off. I couldn't figure out how.